<launch>
  <arg name="bag_file" default="20200430_1f_2.bag" />
  <arg name="map_file" default="20200402_TT_1F.yaml" />
  <include file="$(find autolabor_odom)/launch/autolabor_3d_localization_ndt.launch">
    <arg name="debug" value="true" />
    <arg name="map_file" value="$(find autolabor_map_server)/maps/$(arg map_file)" />
    <arg name="bag_file" value="$(env HOME)/bags/autolabor_bag/$(arg bag_file)" />
    <arg name="viz" value="false" />
  </include>

  <node pkg="autolabor_path" type="path_saver" name="path_saver_node" output="screen">
    <remap from="path_in" to="autolabor_odom" />
    <param name="file_path" type="str" value="$(find autolabor_path)/paths/" />
  </node>

  <node name="rviz" pkg="rviz" type="rviz" required="true"
      args="-d $(find autolabor_path)/rviz/path_saver_3d.rviz" />

</launch>
